int32 cameraIndex
int32 PTZType
int32 speed
int32 time